### DISCUSSION TITLE
Starstruck achievement: Star 4 Star

### DISCUSSION THREAD

**Discussion by Keyrxng** [Schweinepriester/github-profile-achievements#158] [Q&A]
...

**Comment by number0x0001** [comment]
Done. Make sure to give this a star: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by Keyrxng** [reply]
https://github.com/Keyrxng/Interactive-Web3-Resume yo

**Reply by faintout** [reply]
https://github.com/faintout/ThreeRoomDemo  reply

**Reply by melvincwng** [reply]
https://github.com/melvincwng/facebook-clone

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
https://github.com/itspasindu/isipathana-college-demo-site

**Comment by Luscarvalho** [comment]
Return the favor ;)

https://github.com/Luscarvalho/Luscarvalho

**Reply by melvincwng** [reply]
https://github.com/melvincwng/facebook-clone

**Reply by RussellDash332** [reply]
https://github.com/RussellDash332/kenghwee

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Reply by aarongeo1** [reply]
@Luscarvalho done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Denisyasyuchenya** [reply]
https://github.com/Denisyasyuchenya/my-location-app done

**Comment by Ysrbolles** [comment]
Done. Make sure to star my repo https://github.com/Ysrbolles/Darkly-42Network

**Reply by number0x0001** [reply]
Please leave on here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by nel0x** [reply]
@number0x0001 Done. Please leave one at https://github.com/nel0x/arch-installer

**Reply by melvincwng** [reply]
https://github.com/melvincwng/facebook-clone ⭐✌

**Reply by OmarHamdann** [reply]
done Please star here 
https://github.com/OmarHamdann/MERAKI_Academy_Project_5

**Reply by WHAHA-HA** [reply]
Already Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by aarongeo1** [reply]
@nel0x @OmarHamdann done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by Denisyasyuchenya** [reply]
https://github.com/Denisyasyuchenya/my-location-app pls star

**Comment by nel0x** [comment]
All above Done. Please star https://github.com/nel0x/arch-installer

**Reply by melvincwng** [reply]
@nel0x , done ^^

You are almost there! 👍🏻

**Reply by nel0x** [reply]
GG :D

**Reply by RussellDash332** [reply]
Done ^
https://github.com/RussellDash332/kenghwee as well :D

**Reply by WHAHA-HA** [reply]
Done. You can star https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by Denisyasyuchenya** [reply]
https://github.com/Denisyasyuchenya/my-location-app pls star

**Comment by melvincwng** [comment]
For those who are interested to help ⭐:

1. https://github.com/melvincwng/facebook-clone
2. https://github.com/Your-First-Open-Source-Project/Your-First-Open-Source-Project

Thanks In advance, glad to see such a vibrant and inclusive community here on GitHub 😊

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by guexs** [reply]
Done

**Reply by guexs** [reply]
https://github.com/wannderlust/wannderlust give me a star please

**Reply by RussellDash332** [reply]
Done, https://github.com/RussellDash332/kenghwee as well :D

**Reply by RussellDash332** [reply]
> https://github.com/wannderlust/wannderlust give me a star please

Done as well!

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by aarongeo1** [reply]
@melvincwng @guexs @RussellDash332 
done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@melvincwng I left you a star, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by RussellDash332** [comment]
Done, can star this ;)
https://github.com/RussellDash332/kenghwee

**Reply by guexs** [reply]
Done

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by ArielMAJ** [reply]
@RussellDash332 I left you a star, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by bacca87** [comment]
Done

https://github.com/bacca87/great2

**Reply by aarongeo1** [reply]
@bacca87 done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
> Done
> 
> https://github.com/bacca87/great2

Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by Semanteo** [comment]
Please if you can star this :
https://github.com/Semanteo/Genepix-Bot
Or
https://github.com/Semanteo/Github-Timeline

Bonus if you can mark my answer ❤️

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by kshitij-halankar** [reply]
@Semanteo Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by aarongeo1** [reply]
@Semanteo done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by WHAHA-HA** [comment]
Already done!
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
Thank  you.

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by Denisyasyuchenya** [reply]
done https://github.com/Denisyasyuchenya/my-location-app

**Comment by Ashu-Barnwal** [comment]
Already done!
https://github.com/Ashu-Barnwal/Color-Plate
Thank you...

**Reply by kshitij-halankar** [reply]
@Ashu-Barnwal Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by Ashu-Barnwal** [reply]
DONE!

**Reply by aarongeo1** [reply]
@Ashu-Barnwal done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by 3kh0** [comment]
I need an answer, gave you a star

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Reply by aarongeo1** [reply]
@danielr1996 done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by aarongeo1** [comment]
done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by Denisyasyuchenya** [reply]
https://github.com/Denisyasyuchenya/my-location-app
done

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@Keyrxng I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Comment by LauraBigoni** [comment]
Please help me star my repo: https://github.com/LauraBigoni/WeatherApp
I will star you back, just leave the link to your repo too. Thank you guys

**Reply by itspasindu** [reply]
> Please help me star my repo: https://github.com/LauraBigoni/WeatherApp I will star you back, just leave the link to your repo too. Thank you guys

Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by itspasindu** [comment]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by LauraBigoni** [reply]
starred 🌟 here's mine https://github.com/LauraBigoni/WeatherApp

**Reply by itspasindu** [reply]
done

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by DevElkami** [comment]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption (MAUI and Winform).

I would appreciate it if you could check it out and give it a star. Thank you.

**Comment by Denisyasyuchenya** [comment]
https://github.com/Denisyasyuchenya/my-location-app done
